.cargo/config.toml

```diff
@@ -2,5 +2,5 @@
 # In theory, your system GCC and libstdc++ should be able to compile everything here. In practice,
 # you probably want these overrides, so they're enabled by default.
 CXX = "clang++"
-CXXSTDLIB = "libc++"
+# CXXSTDLIB = "libc++"
```

Comment:
We can remove this if it not needed anymore.

Reply:
OK